### ISSUE TITLE
Regarding Model Evaluation

### ISSUE THREAD

**Issue by Rizwan-S** [MahmudulAlam/Automatic-Identification-and-Counting-of-Blood-Cells#18]
Is there any command to evaluate the model.
Specifically, I am looking for:
1. Command to get the accuracy of the model.
2. Command to count the number of cells in the test dataset and compare it with the ground truth
3. Command to get the error values for testing dataset.

**Comment by MahmudulAlam** [comment]
Hi @Rizwan-S 

Sorry for the late response. Command line interface was not implemented for this project. However, what you want is quite easy to implement. You can use the parse_annotation function https://github.com/MahmudulAlam/Automatic-Identification-and-Counting-of-Blood-Cells/blob/75451723be1965f0db566dd868ba49b3911e2567/preprocess/parse_annotation.py#L4 to get the ground truth number of cells and compare it model prediction. For test error evaluation, copy the training loss function and run it for the test data.
